app/client/js/user.service.js

```diff
@@ -0,0 +1,45 @@
+(function() {
+  'use strict';
+
+  angular.module('career_atlas')
+    .factory('UserService', UserService);
+
+  UserService.$inject = ['$http'];
+
+  function UserService() {
+    let token;
+
+    function getToken() {
+      return token;
+    }
+
+    function login(email, password) {
+      $http({
+        url: '',
+        method: 'POST',
```

Comment:
Is login a post? We might need to ask Molly but it seems like we are trying to get that information since we are not creating anything